pkg/sdkserver/localsdk.go

```diff
@@ -125,7 +125,7 @@ func NewLocalSDKServer(filePath string) (*LocalSDKServer, error) {
 
 	go func() {
 		for value := range l.update {
-			logrus.Info("gameserver update received")
+			logrus.Debug("gameserver update received")
```

Comment:
Nit: Not connected to the logLevel, but for consistency with log messages above.
```suggestion
			logrus.Debug("GameServer update received")
```